### PR TITLE
Add k8s car support

### PR DESCRIPTION
**Pull request by devkanro**
## Description
This PR add k8s context support for bullettrain-sh.

I would like to transfer [the k8s car repo](https://github.com/devkanro/bullettrain-go-k8s) to bullettrain-sh organization.

## Features:

- Displaying only when kubectl installed
- Kubernetes context display

**Callword**: `k8s`

**Template variables**:

* `.Icon`: the car's icon
* `.Context`: the Kubernetes context text

**Template colours**:

* `c`: the car's colour

## Car options

| Environment variable                  | Description                                                    | Default value                                   |
|:--------------------------------------|:---------------------------------------------------------------|:------------------------------------------------|
| BULLETTRAIN_CAR_K8S_SHOW               | Whether the car needs to be shown all the time.                | true                                           |
| BULLETTRAIN_CAR_K8S_TEMPLATE           | The car's template.                                            | `{{.Icon \| printf " %s " \| c}}{{.Context \| c}}` |
| BULLETTRAIN_CAR_K8S_PAINT              | Colour override for the car's paint.                           | white+h:yellow                                       |
| BULLETTRAIN_CAR_K8S_SYMBOL_ICON        | Icon displayed on the car.                                     | `⎈`                                             |
| BULLETTRAIN_CAR_K8S_SEPARATOR_PAINT    | Colour override for the car's right hand side separator paint. | Using default painting algorythm.               |
| BULLETTRAIN_CAR_K8S_SEPARATOR_SYMBOL   | Override the car's right hand side separator symbol.           | Using global symbol.                            |
| BULLETTRAIN_CAR_K8S_SEPARATOR_TEMPLATE | Defines the car separator's template.                          | Using global template.                          |

Snipaste_2019-05-29_17-55-29

### CODE REVIEW

**Comment by devkanro** [comment]
It also fixed #45 .

File: defaults.go [hunk 1]
```diff
@@ -15,13 +15,14 @@ import (
 	"github.com/bullettrain-sh/bullettrain-go-php"
 	"github.com/bullettrain-sh/bullettrain-go-python"
 	"github.com/bullettrain-sh/bullettrain-go-ruby"
+	"github.com/devkanro/bullettrain-go-k8s"
```
Comment (vadviktor) [inline defaults.go]: For this PR, since it will end up living on this repo, please change the path from your own name.

Comment (devkanro) [reply defaults.go]: When you accept the repo, I will change it

**Comment by vadviktor** [comment]
Thanks, @devkanro for this PR, really appreciate it :) Though, please update this PR with the rest of the code, I don't want any core plugins to live inside the core. If you really wish to create this as a plugin then it should not be turned on/imported in the core but could be referenced in the README instead.

**Comment by devkanro** [comment]
@vadviktor 
I would like to transfer the [bullettrain-go-k8s](https://github.com/devkanro/bullettrain-go-k8s) to the bullettrain-sh organization, and I am happy if I can join the bullettrain-sh organization.

Another way is bullettrain-sh create a fork of [bullettrain-go-k8s](https://github.com/devkanro/bullettrain-go-k8s).

I will change the import to "github.com/bullettrain-sh/bullettrain-go-k8s" after I transferred to organization or bullettrain-sh has a fork of it.

**Comment by vadviktor** [comment]
@devkanro I don't own the github organization and for me to keep track of this project, for now, PRs are welcome but I'm not going to get more people in just yet.

I'll fork the repo a bit later this week but also going to have this plugin not showing by default as I want to keep the core display small out of the box.

Thanks again for this contribution!

**Comment by vadviktor** [comment]
It is now merged locally.

**Pull request closed without merging**